webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/MeshFileService.scala

```diff
@@ -147,7 +145,20 @@ case class MeshSegmentInfo(chunkShape: Vec3Float, gridOrigin: Vec3Float, lods: L
 object MeshSegmentInfo {
   implicit val jsonFormat: OFormat[MeshSegmentInfo] = Json.format[MeshSegmentInfo]
 }
-case class WebknossosSegmentInfo(transform: Array[Array[Double]], meshFormat: String, chunks: MeshSegmentInfo)
+case class WebknossosSegmentInfo(transform: Array[Array[Double]], meshFormat: String, chunks: MeshSegmentInfo) {
+  def merge(that: WebknossosSegmentInfo): WebknossosSegmentInfo =
+    // assume that transform, meshFormat, chunkShape, gridOrigin, scale and vertexOffset are the same
```

Comment:
It looks like two things are named chunks here, the `MeshSegmentInfo` object and also the `lod.chunks` items. Can we rename the outer one?

Only the (inner)chunks are merged here, right? So maybe calling the method `merge_chunks` makes the assumption more clear when being called as well.

Reply:
These data structures are a bit hard to name as they kind of live between the wk terminology and neuroglancer meshfile manifest terminology. I renamed this to segmentInfo here now, but to be honest, I’m not 100% certain leo’s reasoning for naming choices here :sweat_smile:

Reply:
Fair enough. What do you think about renaming the funtion to `merge_chunks`?

Reply:
Yep, fine for me :+1: